README.md

```diff
@@ -55,4 +55,25 @@ use `$ ps aux` to locate the node process, terminate it, then restart it with `$
 
 
 
-Must include `config.js` and `auth-server/config.php` (not included in repo) to work.
\ No newline at end of file
+Must include `config.js` and `auth-server/config.php` (not included in repo) to work.
+
+
+##Bypass Certificate Authentication
+
+1. install local mongodb: `https://docs.mongodb.com/manual/installation/`
+
+2. open terminal and run: `$ mongod`
+
+3. open another terminal and run: `$ mongo`
+
+4. In the mongo shell, create database "projx": `> use projx`
+
+5. In projx database create "users" collection: `> db.createCollection("users")`
+
+6. Create your own user profile with an email address and full name: `>  db.users.insert({"email":"YOUR_EMAIL_HERE","name":"YOUR_NAME_HERE","isAdmin":true})`. Shutdown mongodb with `> exit`. 
+
+7. Update `devEmail` and `devName` in `config.json` accordingly to the projile you created in step 6. 
+
+8. Change `mongoUri` in `config.json` from `"data"` to `"mongodb://localhost:27017/projx"`. 
+
+9. For development start, run `$ mongod` in one terminal and `$ gulp` in another terminal in the directory of the projx folder. 
```

Comment:
Great documentation - make sure to update the example `config.json` so that it includes `devName`, `devEmail`, `upcomingEventName`, and `upcomingEventDate` as well.